api/urls.py

```diff
@@ -98,7 +98,18 @@ def register_url(self, subclass, viewset):
     url(r'atom/?', views.CreativeWorksAtom(), name='atom'),
     url(r'graph/?', GraphQLView.as_view(graphiql=True)),
     url(r'userinfo/?', ensure_csrf_cookie(views.ShareUserView.as_view()), name='userinfo'),
-    url(r'search/(?!.*_bulk\/?$)(?P<url_bits>.*)', csrf_exempt(views.ElasticSearchView.as_view()), name='search'),
+
+    # only match _count and _search requests
+    url(
+        r'search/(?P<url_bits>(_|[a-zA-Z]+/_)(?:search|count)/{0,1}$)',
```

Comment:
I think this might actually break? `search` and `count` are outside of the `url_bits` capture group. The view will only end up seeing things preceding `search` or `count`.

You'll probably want something closer to this?

`(?P<url_bits>(?:\w+/)?_(?:search|count)/?)$`

* `(?P<url_bits>` - Start a named capture group
* `(?:\w/+)?` - Optionally match one or more alpha-numeric characters followed by a `/`, without capturing that group
* `_(?:search|count)` - Match either `_search` or `_count`
* `/?` - Optionally match a `/`
* `)` - Close the named capture group
* `$` - Matches the end of the input. Required the last character to be either `/`, the `h` from `search`, or the `t` from `count`